### ISSUE TITLE
Allow configuring brightness settings per display

### ISSUE THREAD

**Issue by xPaw** [jacob-pro/solar-screen-brightness#21]
This is a simple and effective app!

I have two different displays, and as a result they require different brightness settings.

I think starting simple should be enough (get devices and display brightness setting for each of them, and store them in config by their device id). Might need to take care of being able to disable per device (or at least support the case when low/high brightness is the same value).

*Per monitor/device brightness settings is mentioned under planned features, just making an issue to provide a use case.*

**Comment by mruac** [comment]
+1 for this, or being able to set minimum brightness per monitor when adjusting for sunset/sunrise brightnesses so that the brightness can stay consistent during the times its not adjusting.

**Comment by cristipopescu1** [comment]
+1 , perhaps the option to turn the app off for specific displays might also be handy

**Comment by jacob-pro** [comment]
There is now partial support for this in the latest release, although you will need to configure it manually.

You will need to edit the config file: `%LOCALAPPDATA\solar-screen-brightness\config.json`

You can set monitor overrides like this:

```
{
  "brightness_day": 80,
  "brightness_night": 60,
  "transition_mins": 40,
  "location": {
    "latitude": 51.50745,
    "longitude": -0.12777
  },
  "overrides": [
    {
      "pattern": "CH89_C34H89xWJ (DP)",
      "key": "device_description",
      "brightness": {
        "brightness_day": 100,
        "brightness_night": 70
      }
    }
  ]
}
```

You can also disable brightness for that device:

```
    {
      "pattern": "CH89_C34H89xWJ (DP)",
      "key": "device_description",
      "brightness": null
    }
```

The keys and patterns can be found by calling: `ssb-cli.exe --once`

`key` can be any of the listed device properties

`pattern` is a match against the value for that key, it can include `*` to indicate a wildcard match.


I will be working on a GUI for this soon to make it more friendly.

Please let me know if this is suitable for you?

**Comment by xPaw** [comment]
Sounds reasonable.

**Comment by jacob-pro** [comment]
This is now available in the 2.1.0 release: https://github.com/jacob-pro/solar-screen-brightness/releases/tag/2.1.0

![image](https://github.com/jacob-pro/solar-screen-brightness/assets/16433349/116d0a11-1cdd-40ce-8d38-e288e7066c12)
